### ISSUE TITLE
Docs: create the first page of the Contributor Hall of Fame. 

### ISSUE THREAD

**Issue by jakepage91** [tailwarden/komiser#1223]
**Is your feature request related to a problem? Please describe.**
Using the Figma design as a base (reach out to @AllieMendes  or @Azanul  for access to the document) create the main Hall of fame page which will hold the profile of the Komiser contributors as well as holding the badges that contributors collect. 

This hall of fame page will be build in the `docs` folder of the Komiser repo. Built using the mintlify framework (which is what the documentation is build on)

As can be seen in the Figma design the `Hall of Fame` will be placed under the `Community & Support` section.

Link to the Figma file: https://www.figma.com/file/1TQ0gUbs0rdmPgeyurWGWw/Komiser-Docs-revamp-(Aug-2023)?type=design&node-id=79%3A459&mode=dev

Screenshot:
<img width="715" alt="Screenshot 2023-11-21 at 15 15 15" src="https://github.com/tailwarden/komiser/assets/38757612/e8050224-b983-4708-8ecd-5dde3a070383">

**Comment by VanshulB** [comment]
Hey @Azanul could you provide me the access to the figma file, I would like to work on this issue.

**Comment by Azanul** [comment]
> Hey @Azanul could you provide me the access to the figma file, I would like to work on this issue. (I've already requested)

Hi @VanshulB, it looks like there's already an unresolved issue assigned to you.
Please revisit this issue once you've resolved any other issues you've been working on.

**Comment by VanshulB** [comment]
Ok @Azanul I will first resolve that issue.

**Comment by Raj100679** [comment]
@Azanul Could you assign this issue to me, I would love to work on this issue. Also I am new to mintlify can you also provide me link to resources, Thanks.

**Comment by Azanul** [comment]
> @Azanul Could you assign this issue to me, I would love to work on this issue. Also I am new to mintlify can you also provide me link to resources, Thanks.

Hi @Raj100679, Welcome to the community!

Please share your email here or over DM in Discord so that I can give you access to the Figma file.
Which resources are you looking for? Also, join our discord server if you have any further questions or need help.

**Comment by Raj100679** [comment]
@raj.kasat10@gmail.com this is email. I will setup the project locally and will dm if I get stuck or confused , Thanks.

**Comment by Raj100679** [comment]
Hey @Azanul , I have. got the invite , Can’t accept it as I am travelling today, will work on it tomorrow

**Comment by Raj100679** [comment]
Hey @jakepage91 , Sorry I am busy with my semester exams, would not be able to work on this. Will start working after year end break, u can un assign me , I will take a new issue later. Thanks

**Comment by jakepage91** [comment]
Perfect! Thanks for update @Raj100679! Best of luck with your exams. And yeah, once you are free let me know which issue you would like to pick up. 
Cheers!

**Comment by Oluwaseyi3** [comment]
Hi @jakepage91 you can assign this to me

**Comment by jakepage91** [comment]
@Oluwaseyi3 legend! all yours.

**Comment by syedbarimanjan** [comment]
@Oluwaseyi3 are you working on it?

**Comment by Oluwaseyi3** [comment]
Yes. I’m trying to link up with Avinesh. The PR will be opened very soon
